letsencrypt/plugins/manual_test.py

```diff
@@ -54,17 +54,10 @@ def test_perform(self, mock_raw_input, mock_verify, mock_stdout, mock_interactio
         resp = self.achalls[0].response(KEY)
         self.assertEqual([resp], self.auth.perform(self.achalls))
         self.assertEqual(1, mock_raw_input.call_count)
-        mock_verify.assert_called_with(
-            self.achalls[0].challb.chall, "foo.com", KEY.public_key(), 8080)
```

Comment:
It is removed from manual_test.py already, which is indicated by the ``-`` sign. The whole PR addresses Kuba's comment, https://github.com/letsencrypt/letsencrypt/pull/1876#issuecomment-164149879, which indicates that instead of testing on the plugin level, we should test on a more generic one.